src/app/app.component.ts

```diff
@@ -16,7 +16,13 @@ const MATERIALS = [
   MatIconModule,
   MatSidenavModule,
 ];
-
+export enum Role {
+  admin = 'ROLE_ADMIN',
+  empHR = 'ROLE_EMPLOYEE_HR',
+  user = 'ROLE_USER',
+  emp = 'ROLE_EMPLOYEE',
+  empMech = 'ROLE_EMPLOYEE_MECHANIC',
+}
```

Comment:
Usually in enums we do uppercase eg. `ADMIN` or `EMP_HR` but feel free to ignore it if you can't be bothered